client/src/components/pages/profileBusiness/index.js

```diff
@@ -0,0 +1,274 @@
+      const countries = Object.keys(json)
+      let data = countries.filter((country) => {
+        return country.toLowerCase().startsWith(value.toLowerCase())
+      })
+      autoCompleteResultCuntry = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return {
+        autoCompleteResultCuntry,
+        country: value
+      }
+    });
+  };
+
+  handleCityChange = value => {
+    const { country } = this.state
+    let autoCompleteResultCity;
+    if (!value) {
+      autoCompleteResultCity = [];
+    } else {
+      let data = json[country].filter((city) => {
+        return city.toLowerCase().includes(value.toLowerCase());
+      })
+      autoCompleteResultCity = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return { autoCompleteResultCity }
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        console.log('Received values of form: ', values);
+      }
+    });
+  };
+
+  render() {
+    const { getFieldDecorator } = this.props.form;
+    const { autoCompleteResultWebsite, autoCompleteResultCuntry, autoCompleteResultCity } = this.state;
+
+    const formItemLayout = {
+      labelCol: {
+        xs: { span: 0 },
+        sm: { span: 0 },
+      },
+      wrapperCol: {
+        xs: { span: 0 },
+        sm: { span: 24 },
+      },
+    };
+    const tailFormItemLayout = {
+      wrapperCol: {
+        xs: {
+          span: 24,
+          offset: 0,
+        },
+        sm: {
+          span: 16,
+          offset: 8,
+        },
+      },
+    };
+
+    const websiteOptions = autoCompleteResultWebsite.map(website => (
+      <AutoCompleteOption key={website}>{website}</AutoCompleteOption>
+    ));
+
+    const CountryOptionss = autoCompleteResultCuntry.map(country => (
+      <AutoCompleteOption key={country}>{country}</AutoCompleteOption>
+    ));
+
+    const cityOptionsss = autoCompleteResultCity.map(city => (
+      <AutoCompleteOption key={city}>{city}</AutoCompleteOption>
+    ));
+
+    return (
+      <Form {...formItemLayout} onSubmit={this.handleSubmit} className='form'>
+
+        <Form.Item label="Type of busines" className='form--item'>
+          {getFieldDecorator("businestype", {
+            rules: [
+              { required: true, message: "Please select your Type of busines!" }
+            ]
+          })(
+            <Select
+              placeholder="Type of busines"
+              onChange={this.handleSelectChange}
+            >
+              {Options(BusinesTypeValues)}
+            </Select>
+          )}
+        </Form.Item>
+
+        <Form.Item label="Orgnisation Name" className='form--item'>
+          {getFieldDecorator('orgnisation', {
+            rules: [
+              {
+                type: 'string',
+                message: 'The Orgnisation Name should be string!',
+              },
+              {
+                whitespace: true,
```

Comment:
@AbdallahAmmar96 I think organization names could have spaces, so you might need to remove this rule.